gateway/1.7/modules/ROOT/pages/flex-troubleshoot-platform-connection.adoc

```diff
@@ -0,0 +1,53 @@
+= Troubleshooting Platform Connection
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images
+
+When configuring Flex Gateway in either mode, it is possible for the Flex Replica's `registration.yaml` to not be present or for the replica to lose connection to Anypoint Platform.
+
+To troubleshoot if a registration file is present and if Flex Gateway can make platform connections, Flex Gateway provides the following command:
+
+[source,ssh]
+----
+flexctl check connections
+----
+
+The `flexctl check connections` first ensures a `registration.yaml` file is present, and then checks if the replica can connect to the URLs defined in the registration file. If the command is successful, it returns the following `info` message:
+
+----
+$ flectl check connections
+[flexctl][info] Connection to platform endpoints successful.
+----
+
+== No Registration File Found
+
+If the `registration.yaml` file is not found, Flex Gateway returns the following error message:
+
+----
+$ flexctl check connections
+Error: no registration config
+----
+
+To troubleshoot an absent registration file, check your Flex Replica configuration folder. To learn more about registering a Flex Gateway, see following section for your relevant mode:
+
+* xref:flex-conn-reg-run.adoc[]
+* xref:flex-local-reg-run.adoc[]
+
+== Failed Connection
+
+If Flex Gateway cannot to a specific URL, it returns the following error message and the status code of the connection request:
+
+----
+$ flexctl check connections
+Error: failed connecting to https://arm-mcm2-service.kstg.msap.io.
+Status: 404
+Response: <html><body><h1>Resource not found</h1></body></html>
+----
```

Comment:
I'm not sure about including this since it's just a test error, not really going to get something like this. I would replace it with a network error like this one, but being very clear that it is an example
```
$ flexctl check connections
Error: failed connecting to https://anypoint.mulesoft.com/: dial tcp 52.22.227.22:443: connect: network is unreachable
```